packages/riipen-ui-docs/src/pages/getting-started/usage/usage.md

```diff
@@ -2,7 +2,7 @@
 
 <p class="description">Get started with React and Riipen-UI in no time.</p>
 
-Material-UI components work in isolation. **They are self-supporting**, and will only
+Riipen-UI components work in isolation. **They are self-supporting**, and will only
```

Comment:
typo

Reply:
LOL. Guess where I stole most of the architecture code from this repo from?